src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/ApplyChangesToWorkspaceContext.cs

```diff
@@ -57,7 +58,10 @@ public void Initialize(IWorkspaceProjectContext context)
             EnsureInitialized();
         }
 
-        public async Task ApplyProjectBuildAsync(IProjectVersionedValue<IProjectSubscriptionUpdate> update, ContextState state, CancellationToken cancellationToken)
+        public async Task ApplyProjectBuildAsync(IProjectVersionedValue<IProjectSubscriptionUpdate> update, 
+            IProjectVersionedValue<IProjectBuildSnapshot> buildSnapshot,
```

Comment:
This value does not need to be versioned, which will simplify calling code slightly.

```suggestion
            IProjectBuildSnapshot buildSnapshot,
```